src/wp-includes/html-api/class-wp-html-token.php

```diff
@@ -36,10 +36,12 @@ class WP_HTML_Token {
 	/**
 	 * Name of node; lowercase names such as "marker" are not HTML elements.
 	 *
-	 * For HTML elements/tags this value should come from {@see WP_HTML_Processor::get_tag}.
+	 * For HTML elements/tags this value should come from `get_tag`
```

Comment:
```suggestion
	 * For HTML elements/tags this value should come from WP_HTML_Processor::get_tag().

Reply:
This makes sense to me, as the method we're referencing here isn't part of this class.

@dmsnell I guess you'll also want to keep [the `@see WP_HTML_Processor::get_tag()` on R43](https://github.com/WordPress/wordpress-develop/pull/4918/files#diff-a2a58890c64ceffde44307c625eefd5fe11162ac07b0e72f2d6792df2c90219bR43) for better IDE integration, and per precedent elsewhere in this patch?

Reply:
if you trace the changes you will note that I reworded this to avoid what felt like an awkward sentence without the inline syntax to delimit it. I added a full `@see` tag below, so I guess things are duplicated now (which is why I removed the class reference, since it sits in full in lines 43 and 44 immediately below it).

https://github.com/WordPress/wordpress-develop/pull/4918/files#diff-a2a58890c64ceffde44307c625eefd5fe11162ac07b0e72f2d6792df2c90219bR43-R44

this is one of those cases where I felt like there wasn't a good option once we removed the PHPDoc syntax. I might try and update this further because now it's weird again with a duplicate reference right underneath this pseudo-reference.